README.md

```diff
@@ -2185,6 +2185,44 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='no-blank-lines-at-start-or-end-of-non-type-scopes'></a>(<a href='#no-blank-lines-at-start-or-end-of-non-type-scopes'>link</a>) **Remove blank lines at the top and bottom of scopes**, excluding type bodies which can optionally include blank lines. [![SwiftFormat: blankLinesAtStartOfScope](https://img.shields.io/badge/SwiftFormat-blankLinesAtStartOfScope-008489.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#blankLinesAtStartOfScope) [![SwiftFormat: blankLinesAtEndOfScope](https://img.shields.io/badge/SwiftFormat-blankLinesAtEndOfScope-008489.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#blankLinesAtEndOfScope)
+
+  <details>
+
+  ```swift
+  // WRONG
+  class Planet {
+
+    func terraform() {
+
+      generateAtmosphere()
+      generateOceans()
+
+    }
+
+  }
+
+  // RIGHT
+  class Planet {
+
+    func terraform() {
+      generateAtmosphere()
+      generateOceans()
+    }
+    
+  }
+
+  // Also fine!
+  class Planet {
+    func terraform() {
+      generateAtmosphere()
+      generateOceans()
+    }
+  }
```

Comment:
Would it be possible to reverse the order of these and include some `MARK`s to justify the extra newline in the case of the leading/trailing whitespace example? To my eye, the first example feels like it has redundant information; i.e. the terraform method is denoted as within the `Planet` scope both through indentation and leading/trailing newlines, which feels out of the spirit of our style guide in my interpretation

Reply:
Sure, I think reversing the examples makes sense -- I definitely prefer not including top/bottom blank lines in type bodies this short